l5/cypress/e2e/todo.cy.ts

```diff
@@ -118,39 +118,39 @@ describe("Level 5 milestone task details Page", () => {
   });
   it("should click and lead to task details page", () => {
     addEntries();
-    cy.get("button").last().click();
```

Comment:
@vikispeaks I am unclear why are these buttons referred to as `.deleteTaskButton` when they don't have `delete` functionality. From what I understand, clicking them should show the task details.

Am I missing something? Also, these conditions are not included in the milestone spec on LMS?

Reply:
The `delete` are for the tests we cover as a part of L4. If the student has completed L4 and is using the same Repo these should work. The reason we have the use for them is we want to check the tasks being added and removed and click on the pending task and validate the details. 

These are part of the milestone spec for previous level so should already be in the repo.

Reply:
The click event to show details is through the Title of the task. This is covered as a part of the lesson implementation already. `cy.get(".text-base").click();` this covers the scenario. And this code available there on the lesson for the students to see.